### ISSUE TITLE
Switch not visible with NS 3.0 on ios

### ISSUE THREAD

**Issue by ivanbuhov** [NativeScript/NativeScript#4175]
_From @sebawita on May 12, 2017 15:22_

When I use `<Switch>` with NS 3.0, the component doesn't get rendered on iOS (it does work fine on Android though), however there seems to be an empty space where the UI component should be.
```
  <Switch
    [(ngModel)]="profile.angularPro">
  </Switch>
```

However when I use `horizontalAlignment="left"` the component appears just fine.
```
  <Switch
    [(ngModel)]="profile.angularPro"
    class="switch"
    horizontalAlignment="left">
  </Switch>
```

_Copied from original issue: NativeScript/ios-runtime#764_

**Comment by NickIliev** [comment]
@sebawita thanks for posting this issue -  I can confirm that this is reproducible bug for iOS with nativeScript 3.0.0. We will update the information here once we have released a fix.

Reproduced with NativeScript 3.0.0 in both non-Angular ([here](https://github.com/NickIliev/NativeScript-Issues-2017/tree/master/NativeScript/issue_4175_tsc)) and Angular apps ([here](https://github.com/NickIliev/NativeScript-Issues-2017/tree/master/NativeScript/issue_4175)).

For anyone interested, as a temporary workaround, use the solution provided by @sebawita by setting **horizontalAlignment** to **_left_**, **_center_** or **_right_**

**Comment by vchimev** [comment]
This issue does not reproduce with {N} 2.5. Another workaround is to set `width` property.
TODO: Update related test [page](https://github.com/NativeScript/NativeScript/blob/master/apps/app/ui-tests-app/css/views.xml#L4) by removing *height* and *width* properties when fixing the issue.

**Comment by toddanglin** [comment]
Can further confirm that setting the `width` style property to _any_ value will make the `Switch` appear on iOS. So even settings `width:1;` will make the `Switch` appear as expected.

For a "global" workaround 'til a patch is released, I think you should be able to do this in your `app.css` stylesheet using an element selector:

```css
Switch {
   width: 1;
}
```

**Comment by SvetoslavTsenov** [comment]
The fix is available in tns-core-modules@next

**Comment by dtopuzov** [comment]
Fix will be released in 3.0.1 later this week.
